demo/packages/tabs/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,229 @@
+
+So today, I'm happy to share with you all that ING is open-sourcing it's own core library of Web Components: lion! 🎉
+
+## Why Lion?
+
+Imagine the following fictional scenario:
+
+> Alfred works at getRich.com, and is working on a new in-house application.
+
+### Agnostic
+
+Finding components is hard, and can lead to a lot of frustration.
+
+> Alfred finds a perfect component for his use case, but unfortunately, it's written in a specific JavaScript framework, and GetRich.com uses a different framework, so he's not able to use it.
+
+Additionally, companies have their own visual identities, and overriding fully styled components to make them look like the companies identity, can sometimes be more work than starting styling from scratch.
+
+> Alfred finds a component that offers all the needed functionalities, however looks vastly different from GetRich.com's identity.
+
+### Customize Functionality
+
+Once you've found your perfect component, at some point down the line you may discover that the component lacks some kind of functionality that you would like to achieve.
+
+> Alfred decides to copy over the code, and implement the feature himself, and as a result now has to maintain the entire component as well, which becomes an added burden to the project.
+
+### Accessibility
+
+Your components should be usable by every kind of user. Accessibility is hard to get right, but an essential feature, additionally, accessibility issues can be impossible to fix at a later stage without breaking changes, so its crucial to incorporate accessibility right from the start.
+
+> Somewhere down the line, Alfred finds an accessibility issue with a component he found online. He can't fix this issue in his code, so he asks the maintainers to fix the accessibility issue, who tell him it's not possible to fix it without doing a breaking change.
+
+Or, alternatively
+
+> Alfred writes his own component, and the accessibility issue is hard to fix because of the way he initially wrote his HTML structure, and will also lead to a breaking change.
+
+### Summary
+
+Does Alfred's story feel relatable?
+
+It highlights multiple common issues in modern web development:
+- Finding and adding a dependency is a big deal
+- Picking something because of looks along might not always be the best choice
+- Adjusting custom behavior/stylings can be tough to maintain
+- If a package is popular it doesn't mean it has good accessibility/performance
+- UI components are hard to get _right_
+
+### What can we do to tackle these issues?
+
+🤔 Imagine having components that purely focus on functionality, and leave the design up to you
+🤔 Imagine that these components come with great accessibility and performance
+🤔 Imagine these components offer great extendability and flexibility
+
+Now stop imagining, and look at lion 🦁
```

Comment:
Tiny nitpick from me, but sometimes it's mentioned as `lion` and sometimes as `Lion`